bin/node-template/pallets/template/Cargo.toml

```diff
@@ -13,10 +13,12 @@ repository = "https://github.com/substrate-developer-hub/substrate-node-template
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = [
+codec = { package = "parity-scale-codec", version = "3", default-features = false, features = [
+	"derive",
+] }
```

Comment:
Can we not apply the Rust coding styles to TOML files? I find them to be highly inappropriate, and someone else almost always comes in and changes them back anyway.

Reply:
It is a TOML plugin that does this to reduce the line length, I guess. I have no real preference, just hate to cause or do manual work.